ui/src/chat/ChatThread/ChatThread.tsx

```diff
@@ -191,7 +191,7 @@ export default function ChatThread() {
           <ChatScroller
             key={idTime}
             messages={orderedReplies || []}
-            whom={flag}
```

Comment:
I had intentionally changed these back to whom/flags because we were using them inconsistently. chatinfo and other places expect flags not nests, curious where we expect whoms to be nests

Reply:
In the markRead call in ReplyMessage